### DISCUSSION TITLE
Improved simulation-based calibration paper soon!

### DISCUSSION THREAD

**Discussion by hyunjimoon** [Data4DM/BayesSD#57] [brain belief 🟩]
Summary is as follows and entire paper will be shared soon.

1. simulation-based calibration CHECK is the correct name. Any checks using self-consistency property* can be put under this frame. 

2. I was confused about two different checks based on self-consistency property. A. rank-uniformity and B. distribution equivalence of prior and data-averaged posterior. Hazhir's point-mass prior example explains why check B cannot be stronger than check A. 

3. Check A never detecting 'estimator that always returns prior distribution as posterior' was a BIG problem which new SBC check allows users to in principle detect any possible issue with the posterior. This is made possible by including additional data-dependent test (e.g. likelihood) quantities when running SBC.

* in short is an asymptotic $\tilde{\theta}, \theta$ symmetry on joint space of three: parameter chosen to be estimated $\tilde{\theta}$ , synthetic data $y$, estimated parameter by approximator in the form of samples $\theta$.
